### PR TITLE
Cart and wishlist

### PR DESCRIPTION
**Pull request by omkarpatke**
# Cart and Wishlist

**Screenshots Preview of Cart and Wishlist**
Cart-ecommerce
wishlist

**Checks:**
- Cart.jsx
- Cart.css
- Wishlist.jsx
- Wishlist.css

### CODE REVIEW

File: src/pages/ProductListingPage/ProductListing.jsx [hunk 1]
```diff
@@ -1,8 +1,140 @@
...
+               
+                <input type="range" name="cycle-range" id="cycle-range" min='1000' max='30000' />
+           </div>
+           
+
+           <div className="filter sort-filter">
+               <h3>Sort</h3>
+               <div className="radio-input">
+                <input type="radio" name="sort"/>
+                <label for="low-to-high">low-to-high</label>
```
Comment (adidoshi) [inline src/pages/ProductListingPage/ProductListing.jsx:25]: Here ig you need to add an id attribute to input element so the label click would work & same for checkbox.

File: src/pages/HomePage/Home.jsx [hunk 2]
```diff
@@ -1,10 +1,52 @@
...
+                    <Link to='/products'>
+                        <img className="top-selling-cycle-img" src="https://trackandtrail.in/sites/all/themes/adaptivetheme/at_subtheme/images/cycle-guide-banner.png" alt="guide-cycle-img"/>
+                    </Link>
+                </div>
+    
+                <div className="guide-info">
+                    <h2 className="guide-heading">LET US<span><h1>HELP</h1></span> YOU PICK </h2>
+                    <h4 className="guide-desc">Our cycle guide can help you narrow 
+                        your search.</h4>
+                    <Link to='/products' className="btn link-btn">Get Started</Link>
```
Comment (adidoshi) [inline src/pages/HomePage/Home.jsx:25]: There is an alignment issue with your 'Get Started' button, you can check once.

File: src/components/NavBar/Navbar.jsx [hunk 3]
```diff
@@ -1,8 +1,43 @@
...
+    </div>
+    <nav className="navbar">
+        <div className="nav-brand">
+        <Link to='/products' className='nav-brand-name'> Faster Cycle Store </Link> 
+        <Link to=''><img src={NavLogo} alt="brand-img" className="nav-brand-img" /></Link> 
+        </div>
+       <div className="nav-contents">
+        <div className="search-bar">
+            <input type="text" placeholder="Search" className="search-input"/>
+        </div>
```
Comment (adidoshi) [inline src/components/NavBar/Navbar.jsx]: Just a suggestion, here you can have `<input type="search" />`

**Review by adidoshi — APPROVED** [review]
Rest the UI looks clean!!

File: src/pages/ProductListingPage/ProductListing.jsx (hunk 1 above)
Comment (omkarpatke) [reply src/pages/ProductListingPage/ProductListing.jsx:25]: Done

File: src/components/NavBar/Navbar.jsx (hunk 3 above)
Comment (omkarpatke) [reply src/components/NavBar/Navbar.jsx]: ok, I Will look on that.

**Pull request merged**